aces/Loris
# [Tools] Prototype of selective data extraction and importing to open datasets

### Brief summary of changes

Prototype for data extraction and importing. Must have a mapping CSV file in order for this script to work. This has the format:

> OLDPSCID, NEWPSCID, NEWCANDID

#### To Test
* Run `php tools/dataExtractor.php candidate <column>`.
    * Only the `candidate` table is supported for now
* This script will generate a csv file `candidate_$column_dataExtract_output.csv`
* Run `php tools/dataImporter.php <mapping.csv> <data.csv>`
* Verify that the UPDATE statements printed by the second script correspond to the data in the source database but with the PSCID changed to the new PSCID value.

More details are in the code comments.
